services/order.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+from django.db import transaction
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+        tickets = [
+            Ticket(
+                order=order,
+                row=data["row"],
+                seat=data["seat"],
+                movie_session=MovieSession.objects.get(
+                    id=data["movie_session"]
+                )
+            )
+            for data in tickets
+        ]
+        Ticket.objects.bulk_create(tickets)
+
+
+def get_orders(username: str = None) -> QuerySet:

```

Comment:
Indicate a `QuerySet` of what instances you return in the return type annotation